addons/account_payment_term/tests/test_payment_term.py

```diff
@@ -36,6 +36,18 @@ def setUpClass(cls, chart_template_ref=None):
                 }),
             ],
         })
+        cls.pay_term_days_end_of_month_30 = cls.env['account.payment.term'].create({
+            'name': "special case",
```

Comment:
We have two payment term with name special case, maybe we can be more explicit here